src/components/User/Users/components/UserDetails/UserDetails.Request.vue

```diff
@@ -202,4 +272,9 @@ export default {
 .user-details-request__form-actions {
   max-width: 48rem;
 }
+
+.user-request__reset-to-unverified-btn {
+  max-width: 23.5rem;
```

Comment:
is not it _too_ wide? maybe something like 15rem?

Reply:
Updated